client/my-sites/customer-home/cards/tasks/earn-features/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'my-sites/customer-home/cards/tasks/task';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { TASK_EARN_FEATURES } from 'my-sites/customer-home/cards/constants';
+import earnIllustration from 'assets/images/customer-home/illustration--task-earn.svg';
+
+const EarnFeatures = ( { siteSlug } ) => {
+	const translate = useTranslate();
+
+	return (
+		<Task
+			title={ translate( 'Make money from your website' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Making Money From Your Website' )` *ES Score: 8*